app/javascript/pages/orientation/Carousel.vue

```diff
@@ -0,0 +1,77 @@
+<template>
```

Comment:
Minor: looks like a lot of this file got indented with 4 spaces instead of 2? Would love to get a linter implemented so this kind of thing can be spotted automatically 🔍.